### DISCUSSION TITLE
Squelch tail

### DISCUSSION THREAD

**Discussion by oisact** [egzumer/uv-k5-firmware-custom#237] [General]
I noticed the radio has a loud squelch tail (talking about non-CTCSS operation), and to be honest, I never really used the rig with the stock firmware. Is this just a general weakness with the rig, or is it anything that can be adjusted in the firmware or settings? Thanks.

**Comment by egzumer** [comment]
There is no such setting like "tail loudness", that I can tell you. Other than that, maybe AF filters could be adjusted in the BK4819 RF chip, I have no idea. That is something you have check on your own.

**Reply by oisact** [reply]
It's probably just the hardware delay in processing from when the signal ends and the squelch clamps back down.

**Reply by egzumer** [reply]
I don't see anything out of the ordinary there, but I have only baofeng and quansheng radios and I'm not HAM. If you want to get rid of the tail use STE.

**Reply by gartnl** [reply]
STE only works if the other/both side supports it. STE transmits a short signal when you release the PTT. The other set than mutes for a moment.
Squelch noise is common in almost all sets. Many times it is helpfull because it forms a strong EOT signal.
